app/controllers/renewals_controller.rb

```diff
@@ -73,20 +80,23 @@ def bulk_renewal_flash(response, type)
     def bulk_renewal_summary_flash(response, type)
       return unless response[type].any?
 
-      flash[type] = I18n.t("renew_all_items_summary.#{type}_html", count: response[type].length)
+      flash[type] = t("renew_all_items_summary.#{type}_html", count: response[type].length)
     end
 
     def renewal_prompt(renewal)
       if renewal.respond_to?(:each)
-        title_text = "#{renewal[0].title}, #{renewal[0].call_number}"
-        non_renewal_reason = error_prompt(renewal[1])
-        title_text + non_renewal_reason
+        renewal_obj, error_message = renewal
+
+        non_renewal_reason = error_prompt error_message
+        renewal_obj.bib_summary + (non_renewal_reason || '')
       else
-        "#{renewal.title}, #{renewal.call_number}"
+        renewal.bib_summary
       end
     end
 
     def error_prompt(error_message)
+      return if error_message.empty?
+
       content_tag(:div, "Denied: #{error_message}")
```

Comment:
```suggestion
      content_tag(:p, "Denied: #{error_message}")
```
Sorry I didn't catch this earlier, but, can we wrap this in [a more semantic](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML) paragraph tag instead? Or does that break things?

Reply:
yes, your suggestion is better 👍